### PR TITLE
Esnext ify

### PR DESCRIPTION
**Pull request by sowiecki**
Everything refactored into ES6, and broken up into modules. Could probably use some more refactoring later, but I think this is a good first pass.

Tested that is works, and it's actually running the new bundled code on the office Echo right now.

### CODE REVIEW

File: bin/try-handler.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var handlerModule = require("../dist/bundle.js");
+
+var fakeLambdaContext = {
+  succeed(results) {
+    console.log(results);
```
Comment (dogarfin) [inline bin/try-handler.js:5]: Do we need this?

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export const MEETING_ROOM_AVAILABILITY = '/rest/meetingRoom/availability/FIFTY_ONE';
```
Comment (dogarfin) [inline src/config.js]: This could probably be split up

File: src/handler.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    handleRoomsIntentBookRequest(intent, session, context);
+  } else if ("ExitIntent" === intentName) {
+    handleExitIntentRequest(context);
+  } else {
+    throw "Invalid intent";
+  }
+};
+
+const handleRoomsIntentBookRequest = (intent, context) => {
+  console.log(intent)
```
Comment (dogarfin) [inline src/handler.js:60]: Do we need this too?

File: bin/try-handler.js (hunk 1 above)
Comment (sowiecki) [reply bin/try-handler.js:5]: This was a hacky little way to test that the bundle exports the correct function. I'd like to keep it for now, and put more work on this file in another PR.

File: src/config.js (hunk 2 above)
Comment (sowiecki) [reply src/config.js]: What do you mean? Split by floors?

File: src/handler.js (hunk 3 above)
Comment (sowiecki) [reply src/handler.js:60]: I'm not sure...I don't think so. It was present in the original ES5 code.

File: src/config.js (hunk 2 above)
Comment (dogarfin) [reply src/config.js]: we just have '/rest/meetingRoom' replicated in a lot of places, just to clean things up a little. Doesn't matter that much though :-P

**Comment by dogarfin** [comment]
Looks good to me, i don't see an approve button, so have at it

**Pull request merged**